config/routes.rb

```diff
@@ -1,6 +1,6 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
+  resources :jobs
```

Comment:
Question: what does this line of code do?

And then a follow up, is it doing more than we need it to (creating unnecessary things) and if so how could we restrict it?

Reply:
I believe what resources is doing is setting up 7 default rails routes so you don't have to type them all out 

(e.g like get "/jobs/new", to: "jobs#new")

it's definitely creating unecessary rroutes we don't need right now, like patches etc. We could restrict it by using only

e.g resources: jobs, only[:index,:new] and then increase as we go

Reply:
Awesome 👍